### PR TITLE
added instruction for getting acccess keys by PowerShell commands

### PR DESCRIPTION
**Pull request by alexturkin**
### Problem
Currently we have instructions to obtain access keys only for Unix-based operation systems (MacOS or Linux), which makes it harder for our customers who uses Windows.

### Solution
Add instructions for PowerShell into README file.

PowerShell is chosen as there is no native support for curl-like in Windows Command Prompt (see https://en.wikipedia.org/wiki/PowerShell#Comparison_of_cmdlets_with_similar_commands, row for wget/curl).

### CODE REVIEW

File: cfn/README-GD.md [hunk 1]
```diff
@@ -55,7 +59,7 @@ An example of a successful response is:
 }
 ```
 
-**Important:** If the command returns no output, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
+**Important:** If the command returns error about not having necessary role, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
```
Comment (alexturkin) [inline cfn/README-GD.md]: I've changed this one as it looks a bit outdated: now if role is wrong, there is message

```
The $AL_USERNAME doesn’t have Administrator role. Assigned role is '$ROLES
```

so "no output" sounds misleading to me.

File: cfn/README-GD.md [hunk 2]
```diff
@@ -72,6 +76,65 @@ curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.glo
...
+Make a note of the `access_key_id` and `secret_key` values, which you need when you deploy the CloudFormation template to your AWS account.
+
+**Note:** Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys.
+
+**Type the following command to list access keys:**
+```
+Invoke-RestMethod -Method Get -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys
+```
+
+**Use the selected access_key_id in the following curl command to delete the key:**
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Not a curl command here anymore :-)

File: cfn/README-GD.md [hunk 3]
```diff
@@ -72,6 +76,65 @@ curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.glo
...
+
+```
+Invoke-RestMethod -Method Delete -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
+```
+
+
+#### PowerShell < 3.0
+
+If you don't have PowerShell >= 3.0 installed, we suggest to [upgrade it](https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell#upgrading-existing-windows-powershell) to any version which is >= 3.0 or use command line and windows versions of [curl](https://curl.haxx.se/download.html) and [jq](https://stedolan.github.io/jq/download/) and follow the [instruction for Unix](#unix).
```
Comment (alexturkin) [inline cfn/README-GD.md]: Unfortunately, Windows 7 goes with PS 2.0 installed by default ([proof](https://4sysops.com/archives/powershell-versions-and-their-windows-version/)) which does not support `Invoke-RestMethod` / `Invoke-WebRequest` cmdlet. 

There are ways to work around it ([example on stackoverflow](https://stackoverflow.com/questions/25120703/invoke-webrequest-equivalent-in-powershell-v2)) but it does not really look nice & right way, so I'm not sure if we should really go for that one.

Alternate way which is described here is to upgrade to Windows 7 to have PS 3.0 or further, all links are available on MS website (https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell#upgrading-existing-windows-powershell)

**Comment by kkuzmin** [comment]
@Shaddakim welcome to the review.

File: cfn/README-GD.md (hunk 3 above)
Comment (alexturkin) [reply cfn/README-GD.md]: @mtrybulec @kkuzmin do you guys think it's ok to ask to install PS 3.0, would it be convenient for customers?

File: cfn/README-GD.md [hunk 4]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+(In that example it says that PowerShell 5.1 is installed)
+
+
+#### PowerShell >= 3.0
+
+
+In PowerShell console, please type the following commands, where `<email address>` is your Alert Logic Cloud Insight email address you use to log in, and then enter your password when prompted:
+
+```
+$AL_USERNAME = "<email address>"
```
Comment (kkuzmin) [inline cfn/README-GD.md]: I think we can safely remove it and have username input interactively.

File: cfn/README-GD.md [hunk 5]
```diff
@@ -37,8 +37,12 @@ In order to verify the user has administrator permissions:
 1. Select the user in `AIMS User` section. **Note** you can start typing a name in the search box to find an appropriate user.
 1. Once found check that `user role` list under the `Edit an AIMS User` section has `Administrator` role selected.
 
-The following procedure assumes a Linux-based local machine using [curl](https://curl.haxx.se/) and 
-[jq](https://stedolan.github.io/jq/). For Windows please use command line and windows versions of [curl](https://curl.haxx.se/download.html) and [jq](https://stedolan.github.io/jq/download/).
+
+Depending on operation system, please proceed with the Unix (MacOS, Linux) or with Windows section.
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Maybe something like:
"Depending on the operating system, please go to respective section: Unix (MacOS, Linux) or Windows.

File: cfn/README-GD.md [hunk 6]
```diff
@@ -37,8 +37,12 @@ In order to verify the user has administrator permissions:
...
+
+### Unix
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Perhaps name this section the same as above: "Unix (MacOS, Linux)"?

File: cfn/README-GD.md [hunk 7]
```diff
@@ -37,8 +37,12 @@ In order to verify the user has administrator permissions:
...
+
+The following procedure assumes a Unux-based local machine using [curl](https://curl.haxx.se/) and [jq](https://stedolan.github.io/jq/).
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Spelling: Unux; should be Unix.

File: cfn/README-GD.md (hunk 1 above)
Comment (mtrybulec) [reply cfn/README-GD.md]: Maybe something like:
"If the command returns an error about not having the necessary role, verify..."
?

BTW, is the AIMS acronym known to the user?

File: cfn/README-GD.md [hunk 8]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
 
 ```
 curl -X DELETE -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
 ```
 
+
+### Windows
+
+The instruction depends on which version of PowerShell you have installed.
+You can check it by running PowerShell and typing in:
```
Comment (mtrybulec) [inline cfn/README-GD.md]: I'd drop line 82 and have line 83 say something like this:
"Check the version of PowerShell on your system:"

File: cfn/README-GD.md [hunk 9]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+
+```
+Major  Minor  Build  Revision
+-----  -----  -----  --------
+5      1      16299  98
+
+```
+
+(In that example it says that PowerShell 5.1 is installed)
+
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Maybe:
(In this example, PowerShell 5.1 is installed.)

File: cfn/README-GD.md [hunk 10]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+
+#### PowerShell >= 3.0
+
+
+In PowerShell console, please type the following commands, where `<email address>` is your Alert Logic Cloud Insight email address you use to log in, and then enter your password when prompted:
```
Comment (mtrybulec) [inline cfn/README-GD.md]: "the" missing:
"In the PowerShell console, ..."

File: cfn/README-GD.md [hunk 11]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+```
+An example of a successful response is:
+
+```
+@{access_key_id=712c0b413eef41f6; secret_key=1234567890b3eea8880d292fb31aa96902242a076d3d0e320cc036eb51bf25ad}
+```
+
+**Important:** If the command returns error about not having necessary role, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Same comments as above.

File: cfn/README-GD.md [hunk 12]
```diff
@@ -66,12 +70,71 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+
+Make a note of the `access_key_id` and `secret_key` values, which you need when you deploy the CloudFormation template to your AWS account.
```
Comment (mtrybulec) [inline cfn/README-GD.md]: Maybe
"..., which you need to deploy ..."

File: cfn/README-GD.md (hunk 3 above)
Comment (mtrybulec) [reply cfn/README-GD.md]: Typo: should be Windows, not windows.

I would think it's ok; earlier version are quite old; still, perhaps run this by Product or at the scrum of scrums.

**Comment by alexturkin** [comment]
@Shaddakim PR has been reviewed by team. Please welcome to the second round of review :)

File: cfn/README-GD.md [hunk 13]
```diff
@@ -66,12 +70,69 @@ Make a note of the `access_key_id` and `secret_key` values, which you need when
...
+```
+An example of a successful response is:
+
+```
+@{access_key_id=712c0b413eef41f6; secret_key=1234567890b3eea8880d292fb31aa96902242a076d3d0e320cc036eb51bf25ad}
+```
+
+**Important:** If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
```
Comment (alexturkin) [inline cfn/README-GD.md]: @Shaddakim here and for Unix instruction we refer to AIMS APIs - is that ok to do so, i.e. is the AIMS acronym known to the user?

Comment (Shaddakim) [reply cfn/README-GD.md]: Customers do not know the AIMS acronym.  That will need to be changed to something different.

Comment (Shaddakim) [reply cfn/README-GD.md]: Turns out, I am wrong.  CI customers apparently know AIMS, CD customers don't.  My apologies - AIMS should be OK.

Comment (alexturkin) [reply cfn/README-GD.md]: @Shaddakim thank you! Are you ok with the rest of PR?

Comment (Shaddakim) [reply cfn/README-GD.md]: Go ahead and merge it.  I have some changes as to where certain items will appear in the final version of the entire Readme, but this is good for the moment.

**Pull request merged**